clients/go/gengo.go

```diff
@@ -69,6 +69,12 @@ type WagClient struct {
 
 var _ Client = (*WagClient)(nil)
 
+// TODO this is a starting point
+var defaultTransport = &http.Transport{
+    MaxIdleConns:          512,
+    IdleConnTimeout:       100 * time.Second, // we need to set this to be less than the idle timeout of our ALBs, which is 120 seconds
+}
+
 // New creates a new client. The base path and http transport are configurable.
```

Comment:
If the default transport performs well then I say just go with that and get out of the business of tweaking these parameters

Reply:
I'm pretty split brained on it tbh. 

short term -> if `client.SetTransport(http.DefaultTransport)` works out well we can piece meal it into the services we care about most

long term -> before baking it into wag it would be nice to do a fair assessment of organizational requirements vs what the defaults provide. Probably not too big of a deal to me